backend/btrixcloud/users.py

```diff
@@ -328,6 +328,17 @@ async def format_invite(self, invite):
         if invite.oid:
```

Comment:
We'll also need org ID for the rename endpoint?
```suggestion
        if invite.oid:
            result["oid"] = invite.oid
```